lib/display.ex

```diff
@@ -2,6 +2,8 @@ defmodule Display do
   alias IO.ANSI
   @current_dir File.cwd!
   @no_value :ex_unit_no_meaningful_value
+  @window_width 40
```

Comment:
It may be a bit misleading to call this @window_width. That name makes me think that it somehow dynamically calculates the width of the terminal window. How about just @progress_width or @progress_bar_width?